TypeCobol/Compiler/Symbols/Symbol.cs

```diff
@@ -86,42 +133,285 @@ public string Name
         }
 
         /// <summary>
-        /// Type changed event.
+        /// The target AST node if any
         /// </summary>
-        public event System.EventHandler TypeChanged;
+        public Node TargetNode
+        {
+            get;
+            internal set;
+        }
 
-        Type m_Type;
         /// <summary>
-        /// Symbol's type
+        /// Name used for an Indexed Name
+        /// </summary>
+        public virtual string IndexedName => Name;
```

Comment:
What is the purpose of IndexedName ?
It seems it's only used by IndexSymbol. But now Owner == Indexed in IndexSymbol, so it should be the same logic as Symbol base class.

Reply:
Removed all 3 Indexed*Name as they are not used (a06915bae5364c522b05750871eb53939fa7734e).